src/components/ebay-textbox/index.marko

```diff
@@ -12,54 +12,52 @@ static var ignoredAttributes = [
     "floatingLabel"
 ];
 
-$ var isPostfixIcon = input.iconPosition === "postfix" ;
+$ var isPostfixIcon = input.iconPosition === "postfix";
 $ var displayIcon = Boolean(!input.multiline && input.iconTag);
 $ var id = input.id || component.getElId("textbox");
-
-<${input.fluid ? "div" : "span"}
-    style=input.style
-    class=[
-        "textbox",
-        input.class,
-        displayIcon && isPostfixIcon && "textbox--icon-end"
-    ]>
-    <if(input.floatingLabel)>
-        <label
-            for=id
-            class=[
-                "floating-label__label",
-                input.disabled && "floating-label__label--disabled"
-            ]>
-            ${input.floatingLabel}
-        </label>
-    </if>
-
-    <if(displayIcon && !isPostfixIcon)>
-        <${input.iconTag}/>
-    </if>
-
-    <${input.multiline ? "textarea" : "input"}
-        ...processHtmlAttributes(input, ignoredAttributes)
-        id=id
+<${input.floatingLabel && "span"} class="floating-label" data-testid="floating-label">
```

Comment:
Don't add `data-testid`
If we do use `data-testid` it should be passed by tests.

Reply:
Not sure either, I basically just added a conditional wrapper. I guess it must be because it's a wrapper and all code inside shifted